src/Workspaces/Core/Portable/Remote/RemoteFeatureOptions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.Experiments;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Remote
+{
+    internal static class RemoteFeatureOptions
+    {
+        private const string LocalRegistryPath = @"Roslyn\Features\Remote\";
+
+        /// <summary>
+        /// Global switch that determines of OOP can be used for a language feature. Exposed through
```

Comment:
It's a bit weird to have the *same* option exposed in both pages (instead of a per-language option), but I think it's okay.